serverlessworkflow/modules/ROOT/pages/tooling/serverless-logic-web-tools/serverless-logic-web-tools-github-integration.adoc

```diff
@@ -0,0 +1,66 @@
+= GitHub integration
+:compat-mode!:
+// Metadata:
+:description: Serverless Logic Web Tools github integration
+:keywords: kogito, workflow, serverless, editor, web, tools, settings, github, integration
+
+The Serverless Logic Web Tools implements a web version of a git client, allowing users to create, commit, push and pull repositories, making it possible to sync workspaces remotely.
```

Comment:
Is this the name on upstream as well? If not, we can create an attribute in `antora.yml` file and change it accordingly.

Reply:
I'm not sure, can you answer this @ederign?

In any case, I think it's better to create the attribute.